### PR TITLE
feat: add additional storybook stories

### PR DESCRIPTION
**Pull request by pfulton**
## Description
- [X] InlineAlert
- [X] ProgressCircle
- [X] PickerButton
- [X] StatusLight

## How and where has this been tested?
 - **How this was tested:** 
 - **Browser(s) and OS(s) this was tested with:** 

## To-do list

- [ ] If my change impacts other components, I have tested to make sure they don't break.
- [ ] If my change impacts documentation, I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).
- [ ] I have tested these changes in Windows High Contrast mode.

- [ ] This pull request is ready to merge.

### CODE REVIEW

File: components/pickerbutton/stories/pickerbutton.stories.js [hunk 1]
```diff
@@ -16,10 +16,91 @@ export default {
...
+        type: { summary: "boolean" },
+        category: "State",
+      },
+      control: "boolean",
+    },
   },
   args: {
-    rootClass: "spectrum-",
+    rootClass: "spectrum-PickerButton",
+    customClasses: ["spectrum-PickerButton--right"],
```
Comment (castastrophe) [inline components/pickerbutton/stories/pickerbutton.stories.js:94]:
```suggestion
    customClasses: ["spectrum-PickerButton--right"],
```

Oh interesting use-case here. I'd typically used customClasses solely for passing down contextual classes from parent elements but this is an interesting idea.  Would you ever want to offer the direction as a control above?

File: components/pickerbutton/stories/template.js [hunk 2]
```diff
@@ -1,20 +1,67 @@
...
+  isDisabled = false,
+  label,
   // ...globals
 }) => {
+  const labelMarkup =
+    isWorkflowIconOnly === true || isUIIconOnly === true ? '' : html`<span class="${rootClass}-label is-placeholder">${label}</span>`;
+
+  const uiIconMarkup = html`
+    ${Icon({
+      iconName: "Chevron100",
```
Comment (castastrophe) [inline components/pickerbutton/stories/template.js:29]: Don't forget to pass globals down to nested components so they are aware of platform scale and context.

File: components/progresscircle/stories/progresscircle.stories.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  component: "Progresscircle",
+  argTypes: {
+    size: {
+      name: "Size",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["small", "medium", "large"],
```
Comment (castastrophe) [inline components/progresscircle/stories/progresscircle.stories.js:16]: Typically at this level, size is t-shirt sizing. `Small`, `medium`, and `large` here seem more aligned with platform scale. Is that accurate?  If so, you can call the platform scale values from the template inside the `...globals` spread property (it's defined in the parameters file at the storybook level.

**Review by castastrophe — CHANGES_REQUESTED** [review]

File: components/inlinealert/stories/inlinealert.stories.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  description: "In-line alerts display a non-modal message associated with objects in a view. These are often used in form validation, providing a place to aggregate feedback related to multiple fields.",
+  component: "Inlinealert",
+  argTypes: {
+    headerText: {
+      name: "Header Text",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        disable: false,
+        category: "Component",
```
[the remark this answers is not in the document]
Comment (pfulton) [reply components/inlinealert/stories/inlinealert.stories.js:15]: Yeah, I like this, and I do think we can standardize on it. Thanks!

File: components/pickerbutton/stories/pickerbutton.stories.js (hunk 1 above)
Comment (pfulton) [reply components/pickerbutton/stories/pickerbutton.stories.js:94]: Oh, you know: we don't have an example of `--left` on the current docs site, but there's CSS written for it, and it's something that SWC has on their docs. I think this should probably be a knob, and the class can toggle between the two directions, with `--right` being the default. I'll make that change.

File: components/progresscircle/stories/progresscircle.stories.js (hunk 3 above)
Comment (pfulton) [reply components/progresscircle/stories/progresscircle.stories.js:16]: > Typically at this level, size is t-shirt sizing.

Yeah, that makes sense.

> Small, medium, and large here seem more aligned with platform scale. Is that accurate?

It's interesting: I think these are more akin to t-shirt sizes, but without an extra-large size. I was double-checking against the [design spec](https://spectrum.adobe.com/page/progress-circle/#Design-checklist) (see "all platform scales"). What do you think? Leave as is?

Comment (castastrophe) [reply components/progresscircle/stories/progresscircle.stories.js:16]: Oh interesting yeah - I think you're right. Should we use `s`, `m`, `l` to align with the other size values in storybook?

File: components/inlinealert/stories/inlinealert.stories.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+      control: { type: "text" },
+    },
+    variant: {
+      name: "Variants",
+      type: { name: "string" },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["neutral", "info", "positive", "notice", "negative", "closable"],
```
Comment (castastrophe) [inline components/inlinealert/stories/inlinealert.stories.js:36]: Is closable in the same category as the other variants? I'm wondering if you can have a closable alert that is also neutral.

File: components/progresscircle/stories/progresscircle.stories.js (hunk 3 above)
Comment (pfulton) [reply components/progresscircle/stories/progresscircle.stories.js:16]: Yeah, that's good for consistency's sake. I'll make that change.

File: components/inlinealert/stories/inlinealert.stories.js (hunk 5 above)
Comment (pfulton) [reply components/inlinealert/stories/inlinealert.stories.js:36]: Oh, great catch. In our docs site we have "An in-line alert with a close button in the footer. Combine this strategy with any variant" as the definition of Closable, and the actual Spectrum site doesn't have Closable at all...

I'm going to switch it from a Variant to some other kind of option!

**Review by castastrophe — APPROVED** [review]

**Pull request merged**